packages/api-proxy/README.md

```diff
@@ -1,6 +1,6 @@
 # `API PROXY`
 
-> 
+We recommend using npm >= 7.20.6
```

Comment:
I think we need to specify the version of npm. If a user uses lower version of npm, the npm will change package-lock.json a lot. It may panic the user.